README.md

```diff
@@ -293,10 +287,25 @@ installed.
 1. [Build](#build) the changes and push them to some tag
 2. Use the new tag in your workflows:
 
-```diff
--uses: paritytech/pr-custom-review@1
-+uses: paritytech/pr-custom-review@2
-```
+    ```diff
+    -uses: paritytech/pr-custom-review@1
+    +uses: paritytech/pr-custom-review@2
+    ```
+
+### Deployment
+
+1. Create a token with `workflow`, `read:org`, `user` scopes.
+
+    ![Token scopes](./img/token-scopes.png)
+
+    and save it into the repo or orga secrets.
```

Comment:
Specifically it has to be Action secrets, right? `https://github.com/organizations/${ORG}/settings/secrets/actions`

Link to those articles:

https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-an-organization

https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow

Reply:
In the new terminology yeah